tests/installer/default.nix

```diff
@@ -0,0 +1,221 @@
+{ binaryTarballs
+, nixpkgsFor
+}:
+
+let
+
+  installScripts = {
+    install-default = {
+      script = ''
+        tar -xf ./nix.tar.xz
+        mv ./nix-* nix
+        ./nix/install --no-channel-add
+      '';
+    };
+
+    install-force-no-daemon = {
+      script = ''
+        tar -xf ./nix.tar.xz
+        mv ./nix-* nix
+        ./nix/install --no-daemon
+      '';
+    };
+
+    install-force-daemon = {
+      script = ''
+        tar -xf ./nix.tar.xz
+        mv ./nix-* nix
+        ./nix/install --daemon --no-channel-add
+      '';
+    };
+  };
+
+  disableSELinux = "sudo setenforce 0";
+
+  images = {
+
+    /*
+    "ubuntu-14-04" = {
+      image = import <nix/fetchurl.nix> {
+        url = https://app.vagrantup.com/ubuntu/boxes/trusty64/versions/20190514.0.0/providers/virtualbox.box;
+        hash = "sha256-iUUXyRY8iW7DGirb0zwGgf1fRbLA7wimTJKgP7l/OQ8=";
+      };
+      rootDisk = "box-disk1.vmdk";
+      system = "x86_64-linux";
+    };
+    */
+
+    "ubuntu-16-04" = {
+      image = import <nix/fetchurl.nix> {
+        url = https://app.vagrantup.com/generic/boxes/ubuntu1604/versions/4.1.12/providers/libvirt.box;
```

Comment:
Should all these plain URLs be quoted (especially due to RFC45: https://github.com/NixOS/rfcs/pull/45)?

Reply:
I wasn't able to get it working with a plain string: 

```
❯  nix build .#hydraJobs.installerTests.rhel-8.x86_64-linux.install-force-daemon --rebuild -L
warning: Git tree '/home/ana/git/nixos/nix' is dirty
error: some outputs of '/nix/store/m54sb60nb2nnrmfmfp5ai2dakc54ry5c-installer-test-rhel-8-install-force-daemon.drv' are not valid, so checking is not possible
```

Reply:
If you try without `--rebuild`, it should succeed.